lib/constructs/interface.js

```diff
@@ -317,11 +446,11 @@ class Interface {
         return `${value} !== ${unsupportedValue}`;
       }
       return `${O}[impl][utils.supportsPropertyName](${P})`;
-    }
+    };
 
     // "named property visibility algorithm"
     // If `supports` is true then skip the supportsPropertyName check.
-    function namedPropertyVisible(P, O, supports = false) {
+    const namedPropertyVisible = (P, O, supports = false) => { // eslint-disable-line func-style
```

Comment:
Other such functions all use `this` value somewhere, so the linter doesn't complain about it. This one doesn't yet, but it seems odd that the styles are different for functions of the same ilk.

I'll change it of course, just wasn't entirely convinced enabling this style rule for these functions was the best idea.